_inc/lib/core-api/wpcom-endpoints/class-wpcom-rest-api-v2-endpoint-admin-menu.php

```diff
@@ -317,6 +319,30 @@ private function prepare_menu_item_url( $url, $parent_slug = '' ) {
 
 		return $url;
 	}
+
+	/**
+	 * Parses the update count from a given menu item title and removes the associated markup.
```

Comment:
This accurately describes _what_ is happening but coming to this without context it might be good to explain _why_ it's necessary. What's the use case that necessitated this? What issues did it cause and how does this fix it?

Small thing but will make big difference in future if/when we return to this.

Reply:
1f462a444c9a5c0c979124e53712ae0bf3b00e8a